redi-connect-front/src/components/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import _uniqueId from 'lodash/uniqueId'
+import { Form, Element } from 'react-bulma-components'
+import './Checkbox.scss'
+
+interface Props {
+  name: string
+  value: string
+  checked: boolean
+  children: React.ReactNode
+  disabled: boolean
+  className?: string
+  customOnChange?: Function
+}
+
+// introduce proper types at some point
+function Checkbox (props: any) {
```

Comment:
Should be easy to wrap up the `Props` interface and use it here. Also remember that `React.FunctionComponent` provides a built-in definition of `children`.

Reply:
There are a lot of props here and additionally formik props also. This is the reason why I kept my distance from this one and made the comment. Lets tacke things like this in July. 
Deal?

Reply:
Deal. Feel free to spank me if I forget the deal.